internal/exec/exec.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/JoinCAD/graphql-go/trace"
 )
 
+const asyncDisabled = true
```

Comment:
Make this a var, give it a capital first letter, and add a comment explaining what it does. That way, if we want to change behavior we can do it in our application without updating the dependency.